src/session/SessionManager.cpp

```diff
@@ -247,23 +282,43 @@ void SessionManager::addLabel(const Label &l) {
 }
 
 /**
-* Removes a comment from the session_data
+* Removes a breakpoint from the session_data
 * @param address
 */
-void SessionManager::removeComment(edb::address_t address) {
+void SessionManager::removeBreakpoint(edb::address_t address) {
 	QString hexAddressString   = address.toHexString();
-	QVariantList comments_data = sessionData_["comments"].toList();
+    QVariantList breakpoints_data = sessionData_["breakpoints"].toList();
 
-	auto it = std::find_if(comments_data.begin(), comments_data.end(), [&hexAddressString](QVariant entry) {
+    auto it = std::find_if(breakpoints_data.begin(), breakpoints_data.end(), [&hexAddressString](QVariant entry) {
```

Comment:
Can't we have multiple breakpoints at the same address? E.g. with different conditions, different kinds of access... (Maybe not, I don't know.)

Reply:
I agree with you. I think as I did this quickly, I followed the same logic used by comments/label.